_includes/sections/vpn.html

```diff
@@ -75,6 +63,15 @@ <h5><span class="badge badge-info">Extra Functionality</span></h5>
     <p>The ProtonVPN clients have a built-in killswitch to block internet connections outside of the VPN. They also are able to automatically start on boot. ProtonVPN also offers "Tor" servers allowing you to easily connect to onion sites, but we still strongly recommend using the official Tor Browser for this purpose.</p>
```

Comment:
```suggestion
    <p>The ProtonVPN clients have a built-in killswitch to block internet connections outside of the VPN. They also are able to automatically start on boot. ProtonVPN also offers "Tor" servers allowing you to easily connect to onion sites, but we still strongly recommend using <a href="https://www.torproject.org/">the official Tor Browser</a> for this purpose.</p>
```

Why to not make this a link?

Reply:
I see no reason not to.